TanukiKit/Projects.swift

```diff
@@ -184,11 +189,11 @@ public extension TanukiKit {
 // MARK: Router
 
 enum ProjectRouter: Router {
-    case readAuthenticatedProjects(Configuration, String, String)
+    case readAuthenticatedProjects(Configuration, String, String, String, String, String, String, String, String)
```

Comment:
I would love if you could add parameter names to this type. I find no parameter names ok with two or three parameters but this gets a bit out of hand.